js-intro.js

```diff
@@ -11,51 +11,118 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
-
+// Boiling point = 212
+const boilingTemp = (temp) => {
+    // Checks to see if input is of number data type
+    if (typeof temp === "number") {
+        if (temp === 212) {
+            return `${temp} is at boiling point.`;
+        } else if (temp < 212) {
+            return `${temp} is below boiling point.`;
+        } else if (temp > 212) {
+            return `${temp} is above boiling point.`;
+        }
+      // If not of number data type, message appears         
+    } else {
+        return 'Please enter a number and try again.';
+    }
+}
```

Comment:
Amazing job with pseudo-coding! Also good work being thorough with edge-cases, checking if the user has an input error.